consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/OperationDetailsController.scala

```diff
@@ -62,15 +58,17 @@ class OperationDetailsController @Autowired()
   (
     @ApiParam(value = "DataSource uri related to the operation (Output DataSource uri for a Write Operation or one of the Input DataSources uri if it is a Read Operation)")
     @RequestParam("source") source: String,
-    @ApiParam(value = "Id of the application that triggered the operation")
-    @RequestParam("applicationId") applicationId: String
+    @ApiParam(value = "Id of the executionEvent that triggered the operation")
+    @RequestParam("executionEventId") executionEventId: String
```

Comment:
as we agreed let's remove this service method and other means for getting this info

Reply:
Ok, this will be done in my next PR